packages/video_player/video_player_android/pigeons/messages.dart

```diff
@@ -2,44 +2,51 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.9
-
-import 'package:pigeon/pigeon_lib.dart';
+import 'package:pigeon/pigeon.dart';
 
+@ConfigurePigeon(PigeonOptions(
+  dartOut: 'lib/src/messages.g.dart',
+  dartTestOut: 'test/test_api.dart',
+  javaOut: 'android/src/main/java/io/flutter/plugins/videoplayer/Messages.java',
+  javaOptions: JavaOptions(
+    package: 'io.flutter.plugins.videoplayer',
+  ),
+  copyrightHeader: 'pigeons/copyright.txt',
+))
 class TextureMessage {
-  int textureId;
+  int? textureId;
```

Comment:
Shouldn't `textureId` be non-null in all of these, now that that's supported?

Reply:
I mentioned that in my first comment I didn't investigate async or nullability changes.  I just want this PR to be an upgrade of pigeon, no semantics changes.

Reply:
> - // @dart = 2.9

This is a semantics change though.

Reply:
It shouldn't be.  In Pigeon if you switch to NNBD then make all types nullable the semantics are the same, the syntax is changed only.  Pigeon is special since it doesn't allow logic, just declarations.